ckan/lib/helpers.py

```diff
@@ -1201,6 +1201,26 @@ def has_more_facets(facet: str,
     return False
 
 
+@core_helper
+def currently_active_facet(facet: str) -> bool:
+    params_items = request.args.keys()
+    expanded_facet = "_" + facet + "_limit"
+    if facet in params_items or expanded_facet in params_items:
+        return True
+    else:
+        return False
+
+
+@core_helper
+def default_collapse_facets():
+    '''Returns config option for `ckan.default_collapse_facets`.
+    If true, the facets in the secondary will be collapsed by default.
+    If false, the facets will all be open, unless closed by the user.
+    Default is false
+    '''
+    return config.get('ckan.default_collapse_facets') or False
```

Comment:
As this has been declared in the config declaration, it will always have a value, defaulting to False
```suggestion
    return config['ckan.default_collapse_facets']
```